### ISSUE TITLE
Add scrolling effects to "How We Do It" section

### ISSUE THREAD

**Issue by sw00d** [ckcollab/ckcollab#77]
"So when the user first scrolls, it just has the header text, but as they scroll further, the body text fades in"

**Comment by wle8300** [comment]
@sw00d Can you elaborate on this issue?

**Comment by sw00d** [comment]
Yeah I can. This idea is really extremely open. I think we just want to enhance the UX in various ways. But I can push on this a bit this week!

**Comment by wle8300** [comment]
@sw00d Niiiice
